### PR TITLE
account: transaction labels code cleanup

### PR DESCRIPTION
**Pull request by ffranr**
PR https://github.com/lightninglabs/pool/pull/396 was merged prematurely. This PR addresses any remaining review comments.

### CODE REVIEW

File: account/manager.go [hunk 1]
```diff
@@ -908,7 +898,7 @@ func (m *manager) deriveFeeFromTx(ctx context.Context,
 	for _, txIn := range tx.TxIn {
 		spendTx, err := m.locateTxByHash(ctx, txIn.PreviousOutPoint.Hash)
 		if err != nil {
-			return 0, nil
+			return nil, nil
```
Comment (guggero) [inline account/manager.go:901]: Oh, I guess this slipped by in my last review. This should return the error.

**Review by guggero — COMMENTED** [review]
Thanks for the fixes. Looks good, just one small issue.

File: account/manager.go (hunk 1 above)
Comment (ffranr) [reply account/manager.go]: Good catch. Thanks.

**Comment by guggero** [comment]
Trivial diff, going to merge with just a single review.

**Pull request merged**